### DISCUSSION TITLE
Thinking development

### DISCUSSION THREAD

**Discussion by NitinIsFit** [Nit-nit/hazy#8] [General]
@rohanmech yo, [experimental branch](https://github.com/Nit-nit/hazy/tree/Experimental) for contributors lol

**Comment by DevRohanSingh** [comment]
> @rohanmech yo, experimental branch for contributors lol

yeah that's cool! Discussing here sure would be a lot better than on issue thread. Good thinking 🤞

**Reply by NitinIsFit** [reply]
Yeah lol 😅

**Reply by NitinIsFit** [reply]
Thanks

**Reply by NitinIsFit** [reply]
@rohanmech i got my phone fixed 2 weeks ago, i dont know what took me so long to tell you, but i almost forgot about github

we can start working on hazy 👍🙏

**Reply by DevRohanSingh** [reply]
> we can start working on hazy 👍🙏

That's good to hear. How about you contact me on discord (Rohanmech#0105) for further discussion about the specifics.
